.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false,
```

Comment:
totally fine w/ the semicolon change here though.  most languages nowadays don't require them.  only language i use now that requires them is php

Reply:
It's useless screen clutter.